apps/ledger-live-desktop/tests/models/Modal.ts

```diff
@@ -83,4 +103,45 @@ export class Modal {
   async chooseStakeSupportLink(stakeProvider: string) {
     await this.stakeProviderSupportLink(stakeProvider).dispatchEvent("click");
   }
+
+  async getTitleProvider() {
+    await this.titleProvider.waitFor();
+    const provider = await this.titleProvider.textContent();
+    return provider;
+  }
+
+  async continueDelegate() {
+    await this.delegateContinueButton.click();
+  }
+
+  async getSpendableBannerValue() {
+    const amountValue = await this.spendableBanner.textContent();
+    return parseInt(amountValue!.replace(/[^0-9.]/g, ""));
```

Comment:
Worth adding a small comment next to stuff like this, something like `// removing all non numeric characters` (or whatever it's doing).

Also why are you negating the `amountValue!`?

Reply:
comment added, and concerning the other question for the amount value I use the ! because of this:
Assure the TypeScript compiler that amountValue will not be null or undefined when calling the replace method.
But as the good practice is to avoid this I remove the !.